src/test/java/seedu/address/model/person/PatientTest.java

```diff
@@ -0,0 +1,85 @@
+package seedu.address.model.person;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_NAME_BOB;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_NRIC_BOB;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_PHONE_BOB;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_TYPE_BOB;
+import static seedu.address.testutil.TypicalPersons.ALICE;
+import static seedu.address.testutil.TypicalPersons.BOB;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.testutil.PersonBuilder;
+
+public class PatientTest {
+
+    @Test
+    public void isSamePerson() {
+        // same object -> returns true
+        assertTrue(ALICE.isSamePerson(ALICE));
+
```

Comment:
since its patient test should we use `patientBuilder`?

Reply:
Do we need the isSamePerson() method for the Patient also? Because the definition for the same person is the same for Patient and normal Person.

Reply:
Yes isSamePerson is needed but when you create the person, create it using a `patientBuilder`? since they are `patient` now. Like check your cases below, all use `PersonBuilder`.

Reply:
Noted. I've rectified it in the commit "Edit PatientTest.java according to PR Review"